Arduino Code/arduino.ino

```diff
@@ -70,9 +74,17 @@ void writeAddress(unsigned int r, unsigned int c, int v) {
   digitalWrite(WE, HIGH);
   digitalWrite(CAS, HIGH);
   digitalWrite(RAS, HIGH);
+  
+  unsigned int long stopWrite = micros();
+  unsigned int long currentTime = stopWrite - startWrite;
```

Comment:
Not sure what's common with Ardiuno code, but I always like to declare variables at the top of the function. Not sure if this has a performance implication or not (probably not).